atlasdb-config/src/main/java/com/palantir/atlasdb/http/AtlasDbHttpClients.java

```diff
@@ -97,9 +117,22 @@ private AtlasDbHttpClients() {
                 .errorDecoder(errorDecoder)
                 .client(client)
                 .retryer(failoverFeignTarget)
+                .options(feignOptions)
                 .target(failoverFeignTarget);
     }
 
+    @VisibleForTesting
+    static <T> T createProxyWithFailoverWithoutBackoff(
+            Optional<SSLSocketFactory> sslSocketFactory, Collection<String> endpointUris, Class<T> type) {
+        Request.Options options = new Request.Options(ONE_MIILISECOND, ONE_MIILISECOND);
```

Comment:
Hmm. I was concerned that this was dangerous - the Feign options are actually _timeouts_, not really backoff related. We cut them down to stop the tests stalling for 10 or 60s on a doomed request, but this seems a bit excessive.

But the tests are passing, so I must be mistaken here. I'm not sure the name is representative of what the method's doing though.